### PR TITLE
QUI-25 maybe this is a good enough solution

### PR DESCRIPTION
**Pull request by boehs**

### CODE REVIEW

File: src/modules/match/match-summary.tsx [hunk 1]
```diff
@@ -64,12 +69,20 @@ export const MatchSummary = () => {
...
+        {isGood ? <>
+          <MatchSummaryFeedback
+            elapsed={elapsed}
+            highscore={highscore}
+            highscores={leaderboard.data.highscores}
+          />
+          {isEligibleForLeaderboard && <Leaderboard data={leaderboard.data} />}
+
+        </> : <>
+          <Heading size={"2xl"}>Too fast!</Heading>
```
Comment (miapolis) [inline src/modules/match/match-summary.tsx:81]: Label it as "Woah! You're too fast!" or something instead

File: src/modules/match/match-summary.tsx [hunk 2]
```diff
@@ -64,12 +69,20 @@ export const MatchSummary = () => {
...
+          <Text>Your time was too fast for our server to process.{
```
Comment (miapolis) [inline src/modules/match/match-summary.tsx:82]: "Your time was too fast to record on our leaderboard."

File: src/modules/match/match-summary.tsx [hunk 3]
```diff
@@ -32,18 +33,22 @@ export const MatchSummary = () => {
     }
   );
 
+  const isGood = elapsed > MATCH_MIN_TIME
```
Comment (miapolis) [inline src/modules/match/match-summary.tsx:36]: Cringe variable name I'd try 'processable' 'isValid' or 'storable'

File: src/modules/match/match-summary.tsx (hunk 1 above)
Comment (miapolis) [reply src/modules/match/match-summary.tsx:81]: Kind of has a more innocent intent if someone legitimately gets under 1.5

**Review by miapolis — CHANGES_REQUESTED** [review]
This solution is perfectly fine, just change some of the wording and we're good to go.

File: src/modules/match/match-summary.tsx (hunk 3 above)
Comment (boehs) [reply src/modules/match/match-summary.tsx:36]: noIsGood

**Review by miapolis — APPROVED** [review]

**Pull request merged**